ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -85,9 +85,10 @@
 
 	private static final By ALLERGIES_LINK = By.id("allergyui-editAllergies");
 
-	private static final By CONDITIONS_LINK = By
-			.cssSelector(".conditions .info-header i.right");
+	private static final By CONDITIONS_LINK = By.cssSelector(".conditions .info-header i.right");
 
+	private static final By ATTACHMENTS_LINK = By.className("icon-share-alt edit-action right");
```

Comment:
hope we didn't miss any thing before right key word

Reply:
> hope we didn't miss any thing before right key word

Hey @HerbertYiga I think i didn't get what you are pointing out, could you give more light on this so that I look into it

Reply:
i was calling for some thing like By.className("icon-share-alt edit-action i.right");

Reply:
> i was calling for some thing like By.className("icon-share-alt edit-action i.right");

Let me look into this

Reply:
@HerbertYiga could you have a look at my inspection here under the _className_ of this feature
![Screenshot at 2021-03-02 08-34-22](https://user-images.githubusercontent.com/38831673/109603041-4b7d0d80-7b32-11eb-84f9-be6bfc602121.png)